### PR TITLE
Add section about using Cachito without package managers to README

### PR DESCRIPTION
**Pull request by brunoapimentel**
CLOUDBLD-9879

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- n/a Code coverage from testing does not decrease and new code is covered
- n/a OpenAPI schema is updated (if applicable)
- n/a DB schema change has corresponding DB migration (if applicable)
- [x] README updated (if worker configuration changed, or if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -954,3 +954,29 @@ Nexus instead. The modified files will be accessible at the
...
+a request without package managers:
+
+```bash
+curl "localhost:8080/api/v1/requests" \
+    -X POST \
+    -H 'content-type: application/json' \
+    -d '{
+          "repo": "https://github.com/cachito-test/cachito-pip-with-deps/",
+          "ref": "1ca07be3001450dbc4f0224e0f763c60353d0f01"
+        }'
```
Comment (chmeliik) [inline README.md:978]: Cachito will "helpfully" default to `"pkg_managers": ["gomod"]` if not specified. The only way to get empty package managers is explicitly `"pkg_managers": []`

Comment (brunoapimentel) [reply README.md]: Now, isn't that a surprise :smile: 
Fixed it.

File: README.md [hunk 2]
```diff
@@ -954,3 +954,33 @@ Nexus instead. The modified files will be accessible at the
 again in a future request, it will use it directly from Nexus rather than downloading it and
 uploading it again. This guarantees that any dependency used for a Cachito request can be used again
 in a future Cachito request.
+
+## Using Cachito Without Package Managers
+
+Cachito can be used without specifying a package manager in a request. In that case, only the source code
+present in the specified commit in a repository will be downloaded and cached.
+
+Even if there are package manager definitions in the source code (such as a `package.json` or a `requests.txt`
```
Comment (chmeliik) [inline README.md:963]:
```suggestion
Even if there are package manager definitions in the source code (such as a `package.json` or a `requirements.txt`
```

**Review by chmeliik — APPROVED** [review]
LGTM, just a typo (requests.txt -> requirements.txt, didn't even notice on the first read)

File: README.md (hunk 2 above)
Comment (brunoapimentel) [reply README.md]: :man_facepalming:

**Review by serg-cymbaluk — APPROVED** [review]

**Comment by taylormadore** [comment]
+1 LGTM

**Pull request merged**